.github/workflows/unidoc.yaml

```diff
@@ -0,0 +1,14 @@
+  name: "Unidoc generation"
```

Comment:
This is great. Given that now we have this job, can you remove the doc generation from [here](https://github.com/delta-io/delta/blob/master/run-tests.py#L73)?